.github/config.yml

```
@@ -0,0 +1,28 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you 
```

```suggestion
  Hi, and welcome to PyNE!  :wave: Thanks for opening your first issue.  We recommend that you 
```
It's always good to add emojis :smile: